makelib/k8s_tools.mk

```diff
@@ -70,6 +70,7 @@ KUTTL := $(TOOLS_HOST_DIR)/kuttl-$(KUTTL_VERSION)
 
 # the version of uptest to use
 UPTEST_VERSION ?= v0.1.0
```

Comment:
Let's also update the default version to the min version available at the new location, e.g., `v0.12.0.

Reply:
Done!